website_outline.html

```diff
@@ -0,0 +1,69 @@
+<html>
```

Comment:
This file should be deleted altogether now that you have one named `index.html`.